src/python/bin/aqueduct

```diff
@@ -246,6 +244,8 @@ def update():
             with open(os.path.join(server_directory, "db/demo.db"), "wb") as f:
                 f.write(requests.get(os.path.join(s3_server_prefix, "db/demo.db")).content)
 
+            update_server_version()
```

Comment:
why do we need this?

Reply:
The latest migration step requires user’s storage config file, but in the bin/aqueduct [here](https://github.com/aqueducthq/aqueduct/blob/main/src/python/bin/aqueduct#L239-L247), we are executing the migration script before downloading that config file from S3. Basically, line 241-244 should go before 239. Does this make sense? @likawind